apps/system/js/bluetooth_transfer.js

```diff
@@ -483,6 +482,7 @@ var BluetoothTransfer = {
       if (transferInfo.received) {
         // Received file can be opened only
         nData.titleL10n = 'transferFinished-receivedSuccessful-title';
+        nData.contentType = transferInfo.contentType;
```

Comment:
Shall we really need to put `contentType` in `notification.data`? We already assign callback function via `nData.onclick = this.openReceivedFile.bind(this, transferInfo);` in line 486. What's the limitation here so that we cannot access `transferInfo` in `openReceivedFile`?

If we have to put the `contentType` in `notification.data`, please also put `fileName` too. And re-check what info we have accessed in `openReceivedFile` callback.